test/libdnf5/rpm/test_nevra.cpp

```diff
@@ -413,3 +413,18 @@ void NevraTest::test_cmp_naevr() {
     std::sort(actual.begin(), actual.end(), libdnf5::rpm::cmp_naevr<TestPackage>);
     CPPUNIT_ASSERT_EQUAL(std::vector<TestPackage>({foo_0_1_4_noarch, foo_0_1_1_1_noarch}), actual);
 }
+
+
+void NevraTest::test_nevra_operator_less() {
+    TestPackage foo_0_1_1_noarch("foo-1-1.noarch");
+    TestPackage foo_1_1_1_noarch("foo-1:1-1.noarch");
+    TestPackage foo_0_2_1_noarch("foo-2-1.noarch");
+    TestPackage foo_0_1_1_1_noarch("foo-1.1-1.noarch");
+    TestPackage foo_0_1_1_s390x("foo-1-1.s390x");
+
+    CPPUNIT_ASSERT(!(foo_0_1_1_noarch < foo_0_1_1_noarch));
```

Comment:
The tests should cover all operators you add. Please add tests for >, <=, and >=.
I think it's not needed to check various cases of being-less-than because the operator is implemented with test_cmp_nevra() which already covers them.